src/converters/sentencetolabels/labelrecovery.py

```diff
@@ -10,16 +11,16 @@ def recover_labels(sentence: str, labels: list[Label]) -> list[Label]:
     for add_labels in [label_exceptive_clauses]:
         labels += add_labels(sentence, labels)
 
-    return labels 
+    return labels
 
 EXCEPTIVE_CLAUSES = ['unless']
```

Comment:
This looks like a candidate to move into the `src.util.constants` file

Reply:
I moved it to `src.util.constants`